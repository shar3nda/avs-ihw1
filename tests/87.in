9 4 -4 1 2 0 10 7 -1 -1