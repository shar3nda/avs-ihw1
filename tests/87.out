-1 5 9 4 -1 -1 4 4 5 8